src/Drivers/nlpSparse_ex6.cpp

```diff
@@ -6,18 +6,18 @@
 
 /* Test with bounds and constraints of all types. For some reason this
  *  example is not very well behaved numerically.
- *  min   sum 1/4* { (x_{i}-1)^4 : i=1,...,n}
+ *  min   sum scal*1/4* { (x_{i}-1)^4 : i=1,...,n}
  *  s.t.
- *            4*x_1 + 2*x_2                     == 10
- *        5<= 2*x_1         + x_3
- *        1<= 2*x_1                 + 0.5*x_i   <= 2*n, for i=4,...,n
+ *             scal * 4*x_1 + 2*x_2                     == scal*10
```

Comment:
what is the role of/why do we need "scal" ?

Reply:
We can easily scale EX6, and then test how gradient scaling works.